client/components/map/index.jsx

```diff
@@ -32,6 +32,7 @@ const map = ({
     zoom={zoom}
     onLeafletMoveend={(ev) => actions.map.getGeoJSON(ev)}
     onLeafletResize={(ev) => actions.map.getGeoJSON(ev)}
+    onLeafletClick={() => actions.map.setActive(undefined)}
```

Comment:
Do we really want to clear active just because they click somewhere else on the map? hide the popup sure... but I think we'd want to leave the active trail...